### DISCUSSION TITLE
How fileMatch work ?

### DISCUSSION THREAD

**Discussion by dimitriBouteille** [renovatebot/renovate#29475] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

I now have several `composer.json` files in my project, here is an example:

 - www/composer.json
 - www/app/code/Test/Foo/composer.json
 - www/app/code/Test/Bar/composer.json
 - www/lib/Happy/composer.json

 I would just like to oberserver the file `www/composer.json`, I added `composer.fileMatch` in the `renovate.json`:

```json
 {
  "enabledManagers": ["composer"],
  "composer": {
    "fileMatch": [
      "www/composer\\.json$"
    ]
  }
}
```

However, when I run renovate, it keeps looking at all `composer.json` files:(

**How can I make renovate use only `www/composer.json` file ?**

**Comment by rarkins** [comment, marked as answer]
You cannot restrict/reduce the default fileMatch patterns. Instead, use ignorePaths if you need to exclude certain files.

**Comment by dimitriBouteille** [comment]
@rarkins Thanks for the quick reply :)
